### ISSUE TITLE
CornerRadius not working

### ISSUE THREAD

**Issue by Unreal852** [AvaloniaUtils/AsyncImageLoader.Avalonia#5]
Hi, i can't get the CornerRadius property to work, changing it doesn't do anything at all

```XAML
<ail:AdvancedImage Grid.Column="0" HorizontalAlignment="Stretch" VerticalAlignment="Stretch"
                                       Stretch="UniformToFill"
                                       Source="{Binding Thumbnail}" CornerRadius="5" />
```

**Comment by SKProCH** [comment]
Hello, @Unreal852 

Seems like `CornerRadius` is the TemplatedControl property...  
And given that I'm not using Image (control), but drawing the picture directly, that might be a problem. I'll try to solve it, and if I succeed, I'll make a fix.

**Comment by SKProCH** [comment]
Should be published as [3.0.0-avalonia11-preview2](https://www.nuget.org/packages/AsyncImageLoader.Avalonia/3.0.0-avalonia11-preview2) version

**Comment by Unreal852** [comment]
That was fast! Thank you
